src/HealthChecks.AzureStorage/HealthChecks.AzureStorage.csproj

```diff
@@ -8,7 +8,8 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Azure.Storage.Blobs" Version="$(AzureStorageBlobs)" />
-    <PackageReference Include="Azure.Storage.Queues" Version="$(AzureStorageQueues)"/>
+    <PackageReference Include="Azure.Storage.Files.Shares" Version="12.8.0" />
```

Comment:
Please move this version inside props file to keep current design.

Reply:
Fixed it